### ISSUE TITLE
Design Intent - Accessibility Feedback - MHV Secure Messaging, Landing Page/Navigation Updates

### ISSUE THREAD

**Issue by briandeconinck** [department-of-veterans-affairs/va.gov-team#91241]
## Next Steps for the VFS team

- [ ] **Assign** this ticket to the team member(s) responsible for addressing feedback provided by Platform.
- [ ] **Comment** on this ticket:
  - If the Platform reviewer has any **Thoughts/Questions** that require responses.
  - When **Must** feedback has been incorporated. As appropriate, link to any other GitHub issues or PRs related to this feedback.
  - When **Should/Consider** feedback has been incorporated, or if any feedback will not be addressed. As appropriate, link to any other GitHub issues or PRs related to this feedback.
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members`.
- [ ] **Close the ticket** when all feedback has been addressed.

## Thoughts/questions

- I'll reiterate what I said in the meeting---tooltips are a huge pain for accessibility, and I was really dreading looking at the Figma files. But this is great! Without question, one of the best tooltip implementations I've ever seen.

## Feedback
Practice areas will document their feedback on the VFS-provided artifacts following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback). Platform Governance reviewers may also provide additional notes that don’t comment on the artifacts themselves but are important for implementation (eg. engineering/coding notes).

- [ ] **Coding notes:** I know this is early in the process, but when it goes to code here's what I'll be looking for in the tooltip:
   - The button that opens the tooltip also closes the tooltip (in addition to the X button).
   - The button that opens the tooltip has `aria-controls` and `aria-expanded` elements that identify its relationship to the tooltip contents and the state of the tooltip (see the additional info and accordion components currently in VADS).
   - When pressing the X button, keyboard focus returns to the button that opens the tooltip.
- [ ] **Should:** I think you can give the tooltip contents a little more breathing room all around --- the X button on its own line rather than having text wrap around it, larger padding around the text, etc. The whole thing feels information-dense, and a little extra white space can help reduce the cognitive load of parsing the content.
- [ ] **Should:** Likewise, I think you can give the tooltip button some breathing room too. In general, we try to avoid floating things to the right, and I think putting the button on its own line left-aligned would still keep it in close enough proximity to what it's helping with that it'll be clear.
- [ ] **Should:** If/when this reaches the point of design system guidance, I would strongly recommend a "one tooltip per page max" rule. Multiple tooltips scattered around a single page could create problems for screen reader users and voice command users (unclear how to differentiate between multiple help buttons).
- [ ] **Consider:** As much as I'm enthusiastic about your tooltip design... I do think it's worth spending some time exploring non-tooltip solutions. Are you sure that there's not a way to make an additional info work? Would a `help` variation for an alert/expandable alert be more appropriate? etc. I don't have answers to that, but I do think it's worth being 100% sure that existing components won't solve your problem before we add something new. This is as much for cognitive load as for anything else --- are the benefits worth the effort required for users to learn/build intuition about a new element?
- [ ] **Must:** As you develop the "Flag this conversation" concept, be careful about color contrast. The yellow color has a [1.51:1 contrast ratio](https://webaim.org/resources/contrastchecker/?fcolor=FACE00&bcolor=FFFFFF) against a white background. For a non-text element like this, WCAG requires a minimum 3:1 contrast ratio, but ideally I'd like to see something in the 7:1 range or better for maximum perceivability.
- [ ] **Should:** For the tertiary tab-like navigation, it's worth making a plan (and communicating it to your engineers) for how they should behave when a user increases the text size on the page. I think it makes sense for the links to wrap onto another line, but you'll want to make sure it wraps in a deliberate way --- how it's aligned, what kind of line spacing there is, etc --- so that it doesn't look haphazard for those users.

## Governance team actions
- [x] Format feedback as individual tasks (check boxes)
- [ ] Assign this ticket to the VFS team member that opened the Slack request
- [ ] Add the VFS team product label
- [ ] Add the VFS team the feature label (if applicable)
- [x] Add the touchpoint labels
- [x] Add the practice area labels
- [ ] Add the Collaboration Cycle initiative milestone

**Comment by underpaid1ntern** [comment]
CC: @emilytaylorux
